cosmos-init/plugins/packages/cosmosc2-tool-dataextractor/src/tools/DataExtractor/DataExtractor.vue

```diff
@@ -713,90 +723,108 @@ export default {
           (100 * (data[0]['time'] - this.startDateTime)) /
             (this.endDateTime - this.startDateTime)
         )
+
+        if (this.bytesReceived > 200000000) {
+          this.bytesReceived = 0
+          this.createFile()
+        }
```

Comment:
Create a new file every 200MB?

Reply:
Yes.  Otherwise the browser uses up tons of RAM.   At this amount it still gets up to about 1 GB.